exonum-java-binding/cryptocurrency-demo/src/main/java/com/exonum/binding/cryptocurrency/transactions/TransactionError.java

```diff
@@ -16,14 +16,14 @@
 
 package com.exonum.binding.cryptocurrency.transactions;
 
-enum TransactionError {
+public enum TransactionError {
```

Comment:
It's the last remaining class in this package — may move to its parent package and keep package-private.